lib/src/models/bluetooth_provisioning_flow_copy.dart

```diff
@@ -7,8 +7,8 @@ class BluetoothProvisioningFlowCopy {
     this.introScreenSubtitle = 'We\'ll walk you through a short setup process to get your device up and running.',
     this.introScreenCtaText = 'Get started',
     // Intro screen tow
-    this.introScreenTwoTurnOnTitle = 'Turn on your VC Box',
-    this.introScreenTwoTurnOnSubtitle = 'Plug in the VC Box and power it on. The light should be slowly flashing blue.',
+    this.introScreenTwoTurnOnTitle = 'Turn on your machine',
+    this.introScreenTwoTurnOnSubtitle = 'Plug in the machine and power it on. The light should be slowly flashing blue.',
```

Comment:
moving to machine for the generic case b/c I don't think we want VC box here either